input/voting_result.json

```diff
@@ -1,37 +1,112 @@
 {
-    "voting":
-        [
-            ["C", "B", "C", "B", "B"],
-            ["A", "D", "D", "D", "A"],
-            ["D", "C", "A", "C", "D"],
-            ["B", "A", "B", "A", "C"]
-        ],
-    "voting2":
-        [
-            ["C", "B", "C", "B", "A"],
-            ["A", "D", "D", "D", "B"],
-            ["D", "C", "A", "C", "D"],
-            ["B", "A", "B", "A", "C"]
-        ],
-    "voting3":
-        [
-            ["C", "B", "C", "B", "B"],
-            ["A", "D", "D", "D", "A"],
-            ["D", "C", "A", "C", "D"],
-            ["B", "A", "B", "A", "C"]
-        ],
-    "voting4":
-        [
-            ["C", "B", "C", "B", "B"],
-            ["A", "D", "D", "D", "A"],
-            ["D", "C", "A", "C", "D"],
-            ["B", "A", "B", "A", "C"]
-        ],
-    "voting5":
```

Comment:
Do not remove this! There are examples in the notebook that use `voting5` to show how strategic voting works.